src/The_Cartridge_Header.md

```diff
@@ -80,34 +80,34 @@ Code | Publisher
 `08` | Capcom
 `13` | Electronic Arts
 `18` | Hudson Soft
-`19` | b-ai
-`20` | kss
-`22` | pow
+`19` | B-AI
+`20` | KSS
+`22` | Pow
 `24` | PCM Complete
-`25` | san-x
-`28` | Kemco Japan
-`29` | seta
+`25` | San-X
+`28` | Kemco
+`29` | SETA
 `30` | Viacom
 `31` | Nintendo
 `32` | Bandai
 `33` | Ocean/Acclaim
 `34` | Konami
 `35` | Hector
 `37` | Taito
-`38` | Hudson
+`38` | Hudson Soft
 `39` | Banpresto
-`41` | Ubi Soft
+`41` | Ubisoft
```

Comment:
Point of style: Ubisoft was known as Ubi Soft prior to 2003 according to [its article on Logopedia](https://logos.fandom.com/wiki/Ubisoft). Should we use the modern spelling or the GB era spelling?

Reply:
That's a good question, I was thinking about that as well. I went through every single company in those two lists and Ubisoft is not the only one that is subject to this issue.

I would argue that, since the name is not stored as a verbatim string in the game binary, it makes sense to refer to the entity with its present name, but I can see the other point of view as well. And it does bother me that this solution is not entirely technically correct, which is the best kind of correct. 😛

Reply:
Given that the company that was licensed and that published the game was what it was back then, I'll advocate for the old style.

Reply:
"Ubisoft (Ubi Soft until 2003-09-09)"